plugins/rapid7_insightvm/komand_rapid7_insightvm/triggers/scan_completion/trigger.py

```diff
@@ -0,0 +1,267 @@
+import insightconnect_plugin_runtime
+import time
+from .schema import ScanCompletionInput, ScanCompletionOutput, Input, Output, Component
+
+# Custom imports below
+import uuid
+from komand_rapid7_insightvm.util import util
+import csv
+import io
+import json
+from typing import List, Union, Dict
+from insightconnect_plugin_runtime.exceptions import PluginException
+from komand_rapid7_insightvm.util.resource_requests import ResourceRequests
+from komand_rapid7_insightvm.util import endpoints
+
+
+class ScanCompletion(insightconnect_plugin_runtime.Trigger):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="scan_completion",
+            description=Component.DESCRIPTION,
+            input=ScanCompletionInput(),
+            output=ScanCompletionOutput(),
+        )
+
+    CACHE_FILE_NAME = f"site_scans_cache_{time.time()}"
```

Comment:
is there any reason that this is a cached file and not a variable 

if you are using the current time in the name then a new cached file will be created on every run and not persist between runs,

using a variable would avoid having to clean up the unused cached filed at the end of the run